win32ss/user/ntuser/winpos.c

```diff
@@ -3163,13 +3163,12 @@ NtUserChildWindowFromPointEx(HWND hwndParent,
  * @implemented
  */
 BOOL APIENTRY
-NtUserEndDeferWindowPosEx(HDWP WinPosInfo,
-                          DWORD Unknown1)
+NtUserEndDeferWindowPosEx(HDWP WinPosInfo, BOOL bAsync)
```

Comment:
One parameter per line as it was before.

Reply:
OK.